utils/CMakeLists.txt

```diff
@@ -1,21 +1,40 @@
 # Create the debufr.c file
 configure_file(${CMAKE_CURRENT_SOURCE_DIR}/debufr.c.in ${CMAKE_CURRENT_BINARY_DIR}/debufr.c @ONLY)
+configure_file(${CMAKE_CURRENT_SOURCE_DIR}/sinv.f.in   ${CMAKE_CURRENT_BINARY_DIR}/sinv.f90 @ONLY)
 
 list(APPEND _utils_srcs
-  binv.f
-  cmpbqm.f
-  readbp.f
-  readmp.f
-  sinv.f
+  binv.f90
+  cmpbqm.f90
+  sinv.f90
   gettab.f90
+  readbp.f90
+  readmp.f90
   split_by_subset.f90
   xbfmg.c
 )
 
+list(APPEND _no_x 
+  binv
+  sinv
+  readbp
+  readmp
+  cmpbqm
+  gettab
+  split_by_subset
+)
+
 foreach(_src ${_utils_srcs})
   get_filename_component(_name "${_src}" NAME_WE)
   get_filename_component(_ext "${_src}" LAST_EXT)
+
   set(_exec "${_name}.x")
+
+  foreach(_nox ${_no_x})
```

Comment:
I like the idea to remove the .x from the executable names.  At some point I'll plan to do the same with the other utilities (debufr, xbfmg, etc.), and then we can just directly set _exec to ${_name} for **all** of the utilities within this file, without continuing to need a _no_x list.  But this is good for now, and again a nice idea!